Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2338,7 +2371,10 @@ - (void)_completeTextManipulationForItems:(NSArray<_WKTextManipulationItem *> *)
         coreTokens.reserveInitialCapacity(wkItem.tokens.count);
         for (_WKTextManipulationToken *wkToken in wkItem.tokens)
             coreTokens.uncheckedAppend(WebCore::TextManipulationToken { coreTextManipulationTokenIdentifierFromString(wkToken.identifier), wkToken.content, std::nullopt });
-        coreItems.uncheckedAppend(WebCore::TextManipulationItem { coreTextManipulationItemIdentifierFromString(wkItem.identifier), WTFMove(coreTokens) });
+        auto identifiers = coreTextManipulationItemIdentifierFromString(wkItem.identifier);
+        if (!identifiers)
+            continue; // FIXME: Generate an error.
```

Comment:
Why `FIXME` and not do it now?

Reply:
Oops, I totally forgot about this FIXME. I'll just pass on initialized itemID / frameID since TextManipulationController checks the validity of these IDs already.